packages/type-definitions/src/types_7.ts

```diff
@@ -8,6 +8,7 @@ export const types7: RegistryTypes = {
   SchemaIdOf: 'IdentifierOf',
   SchemaDigestOf: 'Hash',
   SwarmIdOf: 'IdentifierOf',
+  Address: "AccountId",
```

Comment:
a blanket addition like this may cause an issue. @vatsa287 was getting signing error due to this being there in developer setup. Would be good to test locally once.